packages/ui/src/components/input/input.tsx

```diff
@@ -13,14 +13,19 @@ type Properties = {
   asTextArea?: boolean;
 };
 
+/*235
+236
+238*/
+
+
 export const Input = forwardRef(
   (
     { value, onChange, className, success, error, asTextArea }: Properties,
     reference: ForwardedRef<HTMLInputElement | HTMLTextAreaElement>,
   ) => {
     const properties = {
       className: classes(
-        'w-full resize-none rounded-xl border border-neutral-200 bg-white px-3 py-2 text-body5 text-neutralGreen-900 caret-neutralGreen-900 shadow-input placeholder:text-neutral-600 lg:text-body4',
+        'w-full resize-none min-h-11 rounded-xl border border-secondary bg-white px-3 py-2 text-body5 text-neutralGreen-900 caret-neutralGreen-900 shadow-input placeholder:text-neutral-600 lg:text-body4',
```

Comment:
we can stay with the neutral-200 and don't create multiple aliases for the same color

Reply:
neutral-200 its #e5e5e5 (229,229,229)
on figma it's #ebecee (235,236,238)

Reply:
I changed it